dotnet/src/InternalUtilities/src/Data/VectorStoreRecordPropertyVerification.cs

```diff
@@ -221,4 +246,8 @@ public static void VerifyGenericDataModelDefinitionSupplied(Type recordType, boo
 
         throw new ArgumentException($"A {nameof(VectorStoreRecordDefinition)} must be provided when using '{nameof(VectorStoreGenericDataModel<string>)}'.");
     }
+
+#pragma warning disable CA1823 // Avoid unused private fields. The fields are used in preprocessor directives like NET6_0_OR_GREATER.
```

Comment:
Should we just include this in .NET 6.0 or greater instead of suppressing the error?

Reply:
Thanks, fixed!